public/404/404.php

```diff
@@ -0,0 +1,28 @@
+<?php require_once("../../private/initialize.php"); ?>
+
+<?php
+    
+    $page_title = "Yabe | Browse | Products";
+    $style_sheets = [
+        "/css/common.css",
+        "/css/404.css",
+    ];
+    $scripts = [
+        "/js/common.js",
+    ];
+    
+    include(SHARED_PATH . "/top.php");
+
+?>
+    <body>
+    <div class="container">
+        <div class="first-4">4</div>
+        <img class="far" src="../media/image/yabe_cursor.gif" alt="YABE crying face">
+        <div class="second-4">4</div>
+        <div class="message">SORRY PAGE NOT FOUND
+            <p>Don't cry. We have more interesting features than your crush <a href="../mall/index.php">Here</a> !</p>
```

Comment:
Please use `url_for()` here;

Grammar error: No space between the last word of a sentence and the exclamation mark. E.g
**Right:** Hello this is a sentence!
**Wrong:** ~~Hello this is a sentence !~~